addons/website_slides/static/src/js/public/components/slide_upload_dialog/slide_upload_select_tags.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="website_slides.SlideUploadSelectTags" t-inherit="web.SelectMenu" t-inherit-mode="primary">
+        <xpath expr="//div[hasclass('text-wrap')]" position="inside">
```

Comment:
can we have a better CSS selector ? (I don't like using style classes to target elements)